config/initializers/flickrie.rb

```diff
@@ -0,0 +1,4 @@
+require 'flickrie'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.